README.md

```diff
@@ -3,3 +3,68 @@
 The goal of this wiki is to modernize the information found in the long-standing OSDev wiki found at [wiki.osdev.org](https://wiki.osdev.org) and to decentralize it. It is currently **WIP** and **not ready** for use.
 
 This wiki is not endorsed by, does not have any connection to, and is not related in any way with another wiki found at [wiki.osdev.org](https://wiki.osdev.org).
+
+# Contributing
+
+## Installing dependencies
+
+This wiki is written in Haskell.
+
+The project has some non-Haskell dependencies. To install these, you will need [Python](https://www.python.org/downloads/), [Ruby](https://www.ruby-lang.org/fr/documentation/installation/) and [Rust](https://www.rust-lang.org/tools/install) installed:
+
+- [Pygments](https://github.com/pygments/pygments) - `pip install Pygments`
+- [AsciiDoctor](https://asciidoctor.org/) - `gem install asciidoctor`
+- [asciidoctor-bibtex](https://github.com/asciidoctor/asciidoctor-bibtex) - `gem install asciidoctor-bibtex`
+- [stork](https://stork-search.net/docs/install) - `cargo install stork-search --locked`
+
+## Building and running
+
+Because building the Haskell dependecies takes a long time, we recommend using our Nix binary cache to get all binary dependecies without building. (~10 min setup time)
+
+### With Nix
+
+[Install nix](https://nixos.org/download.html#download-nix). Please use the single-user installation as it's easier to set up.
```

Comment:
Whether to use single user should be up to the user (same idea as before, dealing with dependencies is out of place for us to instruct, and could only serve to cause confusion).